contracts/evm/contracts/xcall/CallService.sol

```diff
@@ -50,13 +56,17 @@ contract CallService is IBSH, ICallService, IFeeManage, Initializable {
     mapping(uint256 => mapping(string => bool)) private pendingResponses;
 
     mapping(string => address) private defaultConnections;
-    bytes private callReply;
-    Types.ProxyRequest private replyState;
 
     address private owner;
     address private adminAddress;
     address payable private feeHandler;
 
+    mapping(uint256 => Types.RollbackData) private rollbacks;
+    mapping(uint256 => Types.ProxyRequestV2) private proxyReqsV2;
+
```

Comment:
dont we need migration ? since we may have older data in `requests`. How is it handled?